nursery_store.go

```diff
@@ -994,6 +869,8 @@ func (ns *nurseryStore) enterCrib(tx *bbolt.Tx, baby *babyOutput) error {
 
 	// Next, retrieve or create the height-channel bucket located in the
 	// height bucket corresponding to the baby output's CLTV expiry height.
+
+	// TODO: Handle late registration.
```

Comment:
is this not critical at this point?

Reply:
It is a pre-existing bug that I wanted to mark with a TODO.